### PR TITLE
fix(useSortedClasses): sort Tailwind v4 arbitrary values

### PR DESCRIPTION
**Pull request by jiwon79**
## Summary

- Added support for sorting arbitrary values by resolving their CSS property sort key.
- Added support for named values with regular modifiers, such as `bg-red-500/50` and `text-lg/8`, without treating them as unknown classes.

## Decisions
- Replaced property-order lookup with `PROPERTY_INDEX` so arbitrary CSS properties can resolve directly to Tailwind's property order.
- Split functional utility branches into `namedBranches` and `arbitraryBranches` so named and arbitrary candidates are resolved through separate paths.
- Split named value types from arbitrary CSS data types. Named branches use Tailwind parser value shapes, while arbitrary branches use CSS AST-based value classification.

> This PR was implemented with assistance from Codex.

### CODE REVIEW

File: .changeset/five-candies-sing.md
Comment (dyc3) [inline .changeset/five-candies-sing.md:1]: no changeset needed, this code isn't user facing yet

File: crates/biome_js_analyze/tests/sort_v4/cases.jsonc [hunk 1]
```diff
@@ -76,9 +76,32 @@
...
-	"flex bg-red-500/50",         // color modifier — Unknown (TODO: opacity)
-	"flex text-lg/8",             // line-height modifier — Unknown (TODO: line-height)
-	"flex w-1/foo",               // value=Number, modifier=Named(non-numeric) → Unknown
+	"flex bg-red-500/50",         // color modifier — base named color sort key
+	"flex text-lg/8",             // line-height modifier — base named text-size sort key
+	"flex w-1/foo",               // non-fraction modifier — base number sort key
+
+	// ─── sort_class_list edge cases ────────────────────────────────
+	// Class-list behavior lives in fixtures; sort_v4.rs unit tests stay
+	// focused on internal branch/key helpers.
```
Comment (dyc3) [inline crates/biome_js_analyze/tests/sort_v4/cases.jsonc]: nit: this comment is kinda useless. it says things just to say them. the agents tend to do this.

**Review by dyc3 — APPROVED** [review]

File: crates/biome_js_analyze/tests/sort_v4/cases.jsonc (hunk 1 above)
Comment (jiwon79) [reply crates/biome_js_analyze/tests/sort_v4/cases.jsonc]: Remove useless comment and changeset at [a172fe0](https://github.com/biomejs/biome/pull/10414/commits/a172fe0575cc9e3c58fc34065aa7a10f3e141829)
